services/cirrus/cirrus.service.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+const Joi = require('joi')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  message: Joi.alternatives()
+    .try(isBuildStatus, Joi.equal('unknown'))
+    .required(),
+}).required()
+const queryParamSchema = Joi.object({
+  task: Joi.string(),
+  script: Joi.string(),
+}).required()
+
+module.exports = class Cirrus extends BaseJsonService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      base: 'cirrus',
+      pattern: '/github/:user/:repo'/:branch*',
```

Comment:
Oops, typo on my end apologies. This needs to be:
```suggestion
      pattern: '/github/:user/:repo/:branch*',
```